client/zeth/cli/zeth_mix.py

```diff
@@ -75,18 +77,27 @@ def mix(
     tx_value: Optional[EtherValue] = None
     if mixer_desc.token:
         tx_value = EtherValue(0)
+    else:
+        tx_value = tx_value or vin_pub
 
-    tx_hash = zeth_client.joinsplit(
+    mix_params = zeth_client.create_mix_parameters(
         wallet.merkle_tree,
         zeth_address.ownership_keypair(),
         eth_address,
-        eth_private_key_data,
         inputs,
         outputs,
         vin_pub,
-        vout_pub,
-        tx_value)
+        vout_pub)
+
+    if show_parameters:
+        print(f"mix_params={mix_params.to_json()}")
```

Comment:
Is this flag used for debug purposes?

Reply:
The original idea was that for docs / demos it would be nice to be able to show the info sent to the contract (i.e. the data visible to the blockchain), but I am sure it will be useful for debugging purposes as well.

Reply:
> The original idea was that for docs / demos it would be nice to be able to show the info sent to the contract (i.e. the data visible to the blockchain

I see, I haven't ran this command (so don't know how it "looks"). I'll play with it shortly.